source/index.md

```diff
@@ -0,0 +1,168 @@
+---
+permalink: index.html
+layout: page
+---
+
+<img src="https://raw.githubusercontent.com/nexusformat/NIAC/master/NeXus_Logo/NeXus_Logo_dark.svg" alt="NeXus Logo" width="500">
+
+About the NeXus Data Format
+===========================
+
+NeXus is a common data format for neutron, x-ray, and muon science. It
+is being developed as an international standard by scientists and
+programmers representing major scientific facilities in order to
+facilitate greater cooperation in the analysis and visualization of
+neutron, x-ray, and muon data.
+
+## Documentation:  
+* Most recent publication to cite:  
+   *J. Appl. Cryst.* (2015). **48**, 301-305 [doi:10.1107/S1600576714027575](https://doi.org/10.1107/S1600576714027575)
+* [User Manual](https://manual.nexusformat.org/user_manual.html):
+  * [Introduction](https://manual.nexusformat.org/introduction.html) to the concepts behind the NeXus data format
+  * [Design](https://manual.nexusformat.org/design.html): The hierarchical design of NeXus files
+  * [NeXus Class Definitions](https://manual.nexusformat.org/classes/index.html): description of each NXDL specification
+    * [base classes](https://manual.nexusformat.org/classes/base_classes/index.html): components that might be used in any NeXus data file
+    * [application definitions](https://manual.nexusformat.org/classes/applications/index.html): layout specifications for a specific purpose
+    * [contributed definitions](https://manual.nexusformat.org/classes/contributed_definitions/index.html): propositions from the community
+  * [Utilities](https://manual.nexusformat.org/utilities.html): Software applications that browse, plot, and analyze NeXus data
+  * [FAQ](https://manual.nexusformat.org/faq.html): Commonly asked questions about NeXus
+  * [Facilities](content/Facilities.html "wikilink") using NeXus
+
+## Discussion and Development:
+* Next Meetings: [Autumn NIAC2024](content/NIAC2024.html "wikilink")
```

Comment:
The current landing page has nothing listed for the next meetings; the item above should be moved to the list of recent meetings